templates/rancher-backup/0/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+backup:
+  environment:
+    CRON_SCHEDULE: ${CRON_SCHEDULE}
+    BACKEND: ${BACKEND}
+    TARGET_PATH: ${TARGET_PATH}
+    BK_FULL_FREQ: ${BK_FULL_FREQ}
+    BK_KEEP_FULL: ${BK_KEEP_FULL}
+    BK_KEEP_FULL_CHAIN: ${BK_KEEP_FULL_CHAIN}
+    VOLUME_SIZE: ${VOLUME_SIZE}
+    RANCHER_API_URL: ${RANCHER_API_URL}
+    RANCHER_API_KEY: ${RANCHER_API_KEY}
+    RANCHER_API_SECRET: ${RANCHER_API_SECRET}
+    DEBUG: false
+    DISABLE_DUMP : ${DISABLE_DUMP}
+    DISABLE_DUMP_RANCHER: ${DISABLE_DUMP_RANCHER}
+    FTP_PASSWORD: ${FTP_PASSWORD}
+    AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
+    AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
+    AZURE_ACCOUNT_NAME: ${AZURE_ACCOUNT_NAME}
+    AZURE_ACCOUNT_KEY: ${AZURE_ACCOUNT_KEY}
+    CLOUDFILES_USERNAME: ${CLOUDFILES_USERNAME}
+    CLOUDFILES_APIKEY: ${CLOUDFILES_APIKEY}
+    DPBX_ACCESS_TOKEN: ${DPBX_ACCESS_TOKEN}
+    GS_ACCESS_KEY_ID: ${GS_ACCESS_KEY_ID}
+    GS_SECRET_ACCESS_KEY: ${GS_SECRET_ACCESS_KEY}
+    GOOGLE_DRIVE_ACCOUNT_KEY: ${GOOGLE_DRIVE_ACCOUNT_KEY}
+    SWIFT_USERNAME: ${SWIFT_USERNAME}
+    SWIFT_PASSWORD: ${SWIFT_PASSWORD}
+    SWIFT_AUTHURL: ${SWIFT_AUTHURL}
+  labels:
+    io.rancher.sidekicks: backup-data
```

Comment:
Add labels to autogenerate RANCHER_API_URL, RANCHER_API_KEY and RANCHER_API_SECRET
++    io.rancher.container.create_agent: true
++    io.rancher.container.agent.role: environment